e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -72,6 +73,21 @@ describe('I18456 Built-in slash commands: common', () => {
             cy.get(`#postMessageText_${postId}`).should('have.text', 'test ¯\\_(ツ)_/¯');
         });
     });
+
+    it('MM-T2345 /me on RHS', () => {
+        loginAndVisitDefaultChannel(user1, testChannelUrl);
+        cy.postMessage(MESSAGES.MEDIUM);
+
+        // # Open RHS (reply thread)
+        cy.clickPostCommentIcon();
+
+        // # type /me test
+        cy.get('#reply_textbox').type('/me test');
+        cy.get('#addCommentButton').click();
```

Comment:
Make sure comment button is visible before clicking

```suggestion
        cy.get('#addCommentButton').should('be.visible').click();
```

Reply:
Cypress should ensure the button isn't hidden and is enabled before clicking. 🤔 💭 
https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Actionability

Could someone from Mattermost confirm?

Reply:
Based on my experience with Cypress, visibility check helps lessen flaky tests, especially with the kind of application we have where the UI relies heavily on async store/state, local storage, REST API calls, etc. It's an implicit wait to make sure that the UI is in ready state (not only a particular element). In this case, I'll add visibility check to `#reply_textbox` before typing since RHS is not open on initial state.
Anyway, I'm good as is (also seeing Joseph already approved).
By the way, in case you're wondering, Joseph is also a staff of Mattermost 😄